### ISSUE TITLE
New Angular 16 not supported by Angular Fire (Peer dependency error)

### ISSUE THREAD

**Issue by sergiotome** [angular/angularfire#3348]
### Version info

Angular: 16.0.0
Firebase: 11.30.0
AngularFire: 7.5.0

### How to reproduce these conditions

I upgraded my project to angular 16 and got a peer dependency error with @angular/fire. Looks like the latest version of Angular Fire expects a version of Angular between 12 and 15, and 16 is not yet supported.

**Steps to set up and reproduce**

Take an existing angular project with Angular Fire 7.5 and upgrade to angular 16. The following error is returned:

![image](https://user-images.githubusercontent.com/16870246/236402395-1c2ede6a-f8a1-4449-8335-b135b9ec4ccc.png)

### Expected behavior

What is the expected behavior? I can upgbrade my project to the latest version of Angular. Angular Fire support the latest version

### Actual behavior

What is the actual behavior? Angular Fire doesn't support the latest version of Angular and I cannot upgrade until a new version of Angular Fire is published with this missing dependency.

**Comment by pascalbe-dev** [comment]
I also came across this during Angular migration. As a workaround, you can manually override the dependencies of `@angular/fire` in your package.json like this:

```
"overrides": {
    "@angular/fire": {
      "@angular/common": "^16.0.0",
      "@angular/core": "^16.0.0",
      "@angular/platform-browser": "^16.0.0",
      "@angular/platform-browser-dynamic": "^16.0.0"
    }
  }
```

This is just a workaround and it assumes, that `@angular/fire` is already compatible with Angular 16. In our case, the whole application still works with that, so we are fine with that. 

Updating the peer dependencies in the `@angular/fire` repo is the better way obviously, but maybe this helps someone :-)

**Comment by armanhadifatah** [comment]
i'm facing the same issue  , this mean one thing google will kill angular

**Comment by jamesdaniels** [comment]
Fixed in 7.6.0, which should be released momentarily

**Comment by jctech1** [comment]
Gracias! @jamesdaniels
